### PR TITLE
sdl must store received languages at get language response from hmi

### PR DESCRIPTION
**Pull request by OHerasym**
Copy commits from [here](https://github.com/CustomSDL/sdl_panasonic/pull/168)

Related:
[APPLINK-20675](https://adc.luxoft.com/jira/browse/APPLINK-20675)

### CODE REVIEW

**Comment by OHerasym** [comment]
Please review: @malirod, @anosach-luxoft , @AByzhynar , @Kozoriz, @VVeremjova , @AMikhniev

File: src/components/application_manager/src/application_manager_impl.cc [hunk 1]
```diff
@@ -333,6 +333,21 @@ void ApplicationManagerImpl::OnApplicationRegistered(ApplicationSharedPtr app) {
   sync_primitives::AutoLock lock(applications_list_lock_);
   const mobile_apis::HMILevel::eType default_level = GetDefaultHmiLevel(app);
   state_ctrl_.OnApplicationRegistered(app, default_level);
+
+  // TODO(AOleynik): Is neccessary to be able to know that registration process
+  // has been completed and default HMI level is set, otherwise policy will
+  // block all the requests/notifications to mobile
```
Comment (Kozoriz) [inline src/components/application_manager/src/application_manager_impl.cc]: please check this comment. if task closed, remove comment.
And maybe we haven`t any reason to duplicate this comment in 2 repo.

**Comment by AMikhniev** [comment]
Reviewed

**Comment by anosach-luxoft** [comment]
@OHerasym please note that branch with CRQ implementation should be named with `feature` prefix

**Pull request merged**